Makefile

```diff
@@ -157,6 +157,10 @@ local-shell: ## Open a command shell in one of the dev containers. ex: make loca
 .PHONY: local-unit-test
 local-unit-test: local-unit-test-backend local-unit-test-processing  local-unit-test-wmg-processing# Run all backend and processing unit tests in the dev environment, with code coverage
 
+.PHONY: redesign-unit-test
```

Comment:
since we're doing a feature branch release anyway, can we delete all the deprecated commands and rename this to just `unit-test `or something?